vcs/file_info.go

```diff
@@ -0,0 +1,23 @@
+package vcs
+
+// ModeSubmodule is an os.FileMode mask indicating that the file is a
+// VCS submodule (e.g., a git submodule).
+const ModeSubmodule = 0160000
```

Comment:
@sqs, what's the reason that this submask is 3 bits rather than a single bit?

I'm seeing a lot of the code here does the following to check if something is a submodule:

```Go
if fi.Mode()&vcs.ModeSubmodule != 0 {
```

But when I tried that, I've run into false positives, which I fixed by doing:

```Go
if fi.Mode()&vcs.ModeSubmodule == vcs.ModeSubmodule {
```

Just checking, do you remember what the context here was?